inference-engine/tests_deprecated/behavior/vpu/myriad_tests/vpu_watchdog_tests.cpp

```diff
@@ -181,7 +181,12 @@ TEST_P(MYRIADWatchdog, watchDogIntervalDefault) {
     {
 
         InferenceEngine::Core core;
-        CNNNetwork network = core.ReadNetwork(GetParam().model_xml_str, Blob::CPtr());
```

Comment:
@gladilov-gleb Please take a look at this change.

As I understand in these tests incorrect model is loading to the plugin and after that it returns error.

But at the current moment I fixed a bug in the ReadNetwork method, and `core.ReadNetwork` doesn't return incorrect model.

I use old reader to read incorrect model.

Reply:
@nikita-kud please take a look

Reply:
LGTM